lib/images.dart

```diff
@@ -0,0 +1,5 @@
+class Images {
+  Images._();
+
+  static const String homeImage = "assets/images/home.png";
```

Comment:
Either use constant, or define a normal class (without static fields), and then declare a constant instance of this class with all the settings you need:
```
class ImagesConfiguration {
  
  ImagesConfiguration ({this.homeImage}) {}
  
  string? homeImage;
}

final Images = Images(homeImage: 'assets/...');
```
(or similar)